src/server/services/tips/predicates.ts

```diff
@@ -169,6 +186,12 @@ export const hasToeristicheVerhuurVergunningen: TipsPredicateFN = (
   );
 };
 
+export const isMarriedOrLivingTogether: TipsPredicateFN = (appState) => {
+  return ['H', 'G']?.includes(
+    appState.BRP?.content?.verbintenis?.soortVerbintenis ?? ''
+  );
+};
+
```

Comment:
Of misschien return `!!appState.BRP?.content?.verbintenis?.soortVerbintenis`. Volgens mij is er geen ander soort verbintenis.